### PR TITLE
Clarify wording of disable/alert device settings and dependency between the two settings

### PR DESCRIPTION
**Pull request by gdepeyrot**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/html/pages/device/edit/device.inc.php [hunk 1]
```diff
@@ -180,9 +180,9 @@
         </div>
     </div>
     <div class="form-group">
-        <label for="disabled" class="col-sm-2 control-label">Disable:</label>
+        <label for="disabled" class="col-sm-2 control-label">Disable polling:</label>
         <div class="col-sm-6">
-          <input name="disabled" type="checkbox" id="disabled" value="1" data-size="small"
+          <input onclick="edit.ignore.disabled=edit.disabled.checked" name="disabled" type="checkbox" id="disabled" value="1" data-size="small"
```
Comment (murrant) [inline includes/html/pages/device/edit/device.inc.php]: your js here is not functioning... If you pull it out of this PR and create a new one once you have it work, we can mere your wording changes now.

Comment (gdepeyrot) [reply includes/html/pages/device/edit/device.inc.php]: Yes, sorry about that. The code works in LibreNMS 1.56 but effectively doesn't work in master...

**Comment by PipoCanaja** [comment]
Bonjour @gdepeyrot 
Thanx for your PR.
Bonne journée !

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-58-release-changelog-november-2019/10224/1

**Pull request merged**